initfiles/componentfiles/configschema/xsd/roxie.xsd

```diff
@@ -188,6 +188,12 @@
             <xs:attribute name="leafCacheMem" type="xs:nonNegativeInteger" hpcc:displayName="Leaf Cache Memory Size (Mb)"  use="optional" default="50" hpcc:tooltip="Size (in Mb) of leaf index page cache"/>
             <xs:attribute name="nodeCachePreload" type="xs:boolean" hpcc:displayName="Enable Node Cache Preload"  use="optional" default="false" hpcc:tooltip="Prefill the node cache with all non-leaf pages from all indexes"/>
             <xs:attribute name="nodeCacheMem" type="xs:nonNegativeInteger" hpcc:displayName="Node Cache Memory Size (Mb)"  use="optional" default="100" hpcc:tooltip="Size (in Mb) of non-leaf index page cache"/>
+            <xs:attribute name="mysqlCacheCheckPeriod" type="xs:nonNegativeInteger" hpcc:displayName="MySQL Cache Check Period(ms)" use="optional" default="10000"
```

Comment:
I thought you were pushing to move away from the defaults in the config and label these as "defaultincode" ?

Reply:
This PR is just to capture the attributes. The full changes for default support will be done as part of a full roxie xsd review.